internal/pkg/exporter/metricssvc/metricssvc.pb.go

```diff
@@ -0,0 +1,484 @@
+//
+//Copyright (c) Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Should it vendor the metrics-exporter repo and use the generated proto files from there?